raisinbread/README.md

```diff
@@ -49,13 +58,14 @@ cat SQL/0000-00-00-schema.sql \
     raisinbread/RB_files/*.sql | mysql
 ```
 
-Note: to empty and delete all RaisinBread tables, use the following command in the 
-main LORIS root directory
-```bash
-cat raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql \
-    SQL/9999-99-99-drop_tables.sql | mysql
+***Important:** Ensure that the above commands (including those to drop tables) were all completed properly.* 
```

Comment:
Users tend to read as they apply commands so by the time they get to this note and if the drop commands did not complete successfully, they most likely already had an issue sourcing the default schemas. I would recommend adding a note like this one around line 40 asking the users to confirm that the drop statements got rid of all the tables in the database before moving on to the sourcing part.

Reply:
Sounds good!